packages/grub/0001-setup-Add-root-device-argument-to-grub-setup.patch

```diff
@@ -1,4 +1,4 @@
-From 9ddb865df4ea2fb79edaf6c3a8b0122796800014 Mon Sep 17 00:00:00 2001
+From fa856e9d6cce8cac8f55bb00392725961c264b8f Mon Sep 17 00:00:00 2001
```

Comment:
Nit: should this patch include a Sign-off comment since you did some refactor?

Reply:
Sure, I can add that.